### ISSUE TITLE
oelint.vars.srcurioptions FP when using inline code blocks

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#142]
False positive of `Fetcher '' is not known` for

```
SRC_URI = "\
           ${@["", "file://init.cfg"][(d.getVar('VIRTUAL-RUNTIME_init_manager') == 'busybox')]} \
"
```
